raiden-dapp/src/components/ErrorDialog.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <v-overlay v-if="description !== ''" class="overlay" absolute="true">
```

Comment:
I think we have a `BlurredOverlay` component already, that could be used whenever an overlay is needed. That way it's more consistent. It's already used in e.g.:

> raiden-dapp/src/components/UpdateSnackbar.vue

Reply:
Yes, that sounds good, I'll take a look at it.